Atomic/atomic.py

```diff
@@ -789,7 +840,6 @@ def get_col_lengths(_images):
             _max_repo, _max_tag = get_col_lengths(_images)
             col_out = "{0:1} {1:" + str(_max_repo) + "} {2:" + str(_max_tag) + \
                       "} {3:14} {4:18} {5:14}"
-            self.write_out("")
```

Comment:
@rhatdan There is a util.write out and a atomic.write out.  They differ.  Should they both exist?